templates/collabora.nomad

```diff
@@ -0,0 +1,85 @@
+{% from '_lib.hcl' import shutdown_delay, group_disk, task_logs, continuous_reschedule with context -%}
+
+job "collabora" {
+  datacenters = ["dc1"]
+  type = "service"
+  priority = 98
+
+  group "collabora" {
+    ${ group_disk() }
+    ${ continuous_reschedule() }
+    network {
```

Comment:
plz keep the member order in more or less the same order as the other templates - network goes at the bottom of the group, alongside other network-y stuff like resources and services.

Also, why is this under group > network and not task > resources > network  + task > configs > port_map? The more consistent we have these, the easier it will be to eventually migrate off this platform into another one.